modules/indexer/stats/indexer.go

```diff
@@ -40,15 +40,18 @@ func populateRepoIndexer() {
 
 	isShutdown := graceful.GetManager().IsShutdown()
 
-	exist, err := db.IsTableNotEmpty("repository")
+	engine := db.GetEngine(db.DefaultContext)
+	tableRepository := new(models.Repository)
+
+	exist, err := engine.Table(tableRepository).Exist()
 	if err != nil {
 		log.Fatal("System error: %v", err)
 	} else if !exist {
 		return
 	}
 
 	var maxRepoID int64
-	if maxRepoID, err = db.GetMaxID("repository"); err != nil {
+	if _, err = engine.Select("MAX(id)").Table(tableRepository).Get(&maxRepoID); err != nil {
```

Comment:
No SQL in modules.

Reply:
The code was there. The code was accessing database directly via db.Xxx.

The new code is just more clear than before.

Adding new packages is not the task or purpose of this PR. This PR just makes a clean up.